install_homebrew.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# A simple Homebrew installation wrapper, to hide the platform-specific stanzas:
+# macOS: https://brew.sh/
+# Linux: https://docs.brew.sh/Homebrew-on-Linux#install
+
+set -e
+umask 022
+
+if [ -z "$USER" ]; then
+  # Note: The Homebrew installation scripts both expect the USER environment
+  # variable to be set, so just check upfront
+  # https://github.com/Linuxbrew/install/issues/48
+  # https://github.com/Linuxbrew/install/pull/49
+  echo "Aborting install! No USER environment variable set"
+  exit 1;
+fi
+
+if [ $(id -u) = 0 ]; then
+  echo "Aborting install! Must not be run as root"
+  exit 2
+fi
+
+if command -v brew >/dev/null 2>&1; then
+  echo "Homebrew already installed 👍"
+  exit
```

Comment:
You probably intend to use a unique exit code here, as per other `exit` calls?

Reply:
My understanding is that the default is `0`, which means no error? I felt like it should be safe (and not an error) to re-run this script?